.github/workflows/update-lavamoat-policies.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: 'lts/*'
+          node-version: '20.14.0'
```

Comment:
This will only impact the version of Node used to enable corepack/install Yarn. The version of Node.js we use to actually run our code is below, the second instance of `setup-node`, which uses the `.nvmrc` file. I think we can leave these lines on `lts/*`, but update `.nvmrc` to run on v20.14.0

Reply:
Ah yeah.  Done in https://github.com/MetaMask/metamask-extension/pull/25534/commits/09f9c4a0c93dd76972d3aed1966e64e3415ff354.

I suppose if we wanted to keep it scoped to the lavamoat stuff, `node-version-file: '.nvmrc'` could be changed to `node-version: '20.14.0'`

Reply:
Hmm. True, good suggestion. I am inclined to update `.nvmrc` instead so that people don't run into this locally as well though. Guto hit this bug just when creating a build. It seems to not be LavaMoat specific, more of a general build system thing.